Bindings/bindings.json

```diff
@@ -619,7 +633,8 @@
                     "resource": "ServiceBus",
                     "required": true,
                     "label": "Notification hub namespace connection",
```

Comment:
Make casing consistent. Should be Notification Hub here and below. Actually the service name is "Notification Hubs" plural. We should probably use that, no?